Model/Dintero.php

```diff
@@ -100,7 +100,7 @@ class Dintero extends AbstractMethod
      *
      * @var bool
      */
-    protected $_canUseInternal = true;
+    protected $_canUseInternal = false;
```

Comment:
What does this do?

Reply:
it disables the usage of dintero when creating orders from admin panel.